packages/components/src/radio/src/Radio.css

```diff
@@ -12,10 +13,11 @@
 
 /* BUTTON */
 .o-ui-radio-button {
-    display: inline-block;
+    align-self: baseline;
     background-color: var(--o-ui-radio-background-color);
     border-radius: var(--hop-shape-circle);
     border: 1px solid var(--o-ui-radio-border-color);
+    margin-top: 1.5px;
```

Comment:
why a margin top? if the positioning of the radio text is done right, this is not needed